auditlog/mixins.py

```diff
@@ -112,6 +116,15 @@ def msg(self, obj):
 
         return mark_safe("".join(msg))
 
+    @admin.display(description="Correlation ID")
+    def cid_url(self, obj):
+        cid = obj.cid
+        if cid:
+            url = self._add_query_parameter("cid", cid)
+            return format_html(
+                '<a href="{}" title="{}">{}</a>', url, self.CID_TITLE, cid
```

Comment:
Please use `f-string` here

Reply:
using `f-string` here defeats the purpose of using `format_html`